pkg/function/client/client_resolver.go

```diff
@@ -0,0 +1,66 @@
+package client
+
+import (
+	"github.com/numaproj/numaflow-go/pkg/function"
+	"google.golang.org/grpc/resolver"
+	"strconv"
+	"strings"
+)
+
+const (
+	custScheme      = "example"
+	custServiceName = "lb.example.grpc.io"
```

Comment:
```suggestion
	custScheme      = "udf"
	custServiceName = "numaflow.numaproj.io"
```

could you please fix?